requirements.txt

```diff
@@ -0,0 +1,23 @@
+# --------
+# requires
+# --------
+
+webob
+
+# --------------
+# tests_requires
+# --------------
+
+requests
+
+# -------
+# develop
+# -------
+
+pytest
```

Comment:
In some other services projects, we use nose rather than pytest for running test.  I hear good things about pytest though!  Is it sufficiently superior to nose that we should do the work of moving other repos over to use it?

Reply:
Definitely, we should. Nose is more or less dead and pytest is in active development.

Reply:
OK, sounds good :-)